content/_data/changelogs/lts.yml

```diff
@@ -7136,6 +7136,654 @@
+    message: |-
+      Fix indistinguishable build scheduling icon when the job is already in-queue.
+
+  - type: bug
+    category: regression
+    pull: 6493
+    issue: 68042
+    authors:
+      - Langer0416
+      - NotMyFault
+      - basil
+    pr_title: "[JENKINS-68042] Help button alignment is wrong if no object is\
+      \ present to place it on (regression in 2.320)"
+    message: |-
+      Fix the position of the help button when it is not directly attached to an object (regression in 2.320).
+
+  - type: bug
+    category: regression
+    pull: 6556
+    issue: 68296
+    authors:
+      - alecharp
+    pr_title: "[JENKINS-68296] Restore actions icon lookup with size"
+    message: |-
+      Restore actions icon size lookup (regression in 2.341).
+
+  lts_changes: # compared to lts_predecessor 2.332.3 (selected by personal review)
+
+  # UI improvements
+
+  - type: major rfe
+    category: major rfe
+    pull: 5923
+    authors:
+      - janfaracik
+    pr_title: Modernise form components
+    message: |-
+      Modernise form components.
+
+  - type: rfe
+    category: rfe
+    pull: 6084
+    issue: 67396
+    authors:
+      - janfaracik
+      - timja
+    pr_title: "[JENKINS-67396] Update link and breadcrumb dropdowns"
+    message: |-
+      Update link and breadcrumb dropdowns.
+
+  - type: rfe
+    category: rfe
+    pull: 6442
+    authors:
+      - NotMyFault
+    pr_title: Use the `jenkins-input` class in more locations
+    message: |-
+      Use modern form inputs in Plugin Manager, login form, and setup wizard.
+
+  - type: rfe
+    category: rfe
+    pull: 5743
+    issue: 68293
+    authors:
+      - jamesD33064
+      - timja
+      - basil
+    pr_title: "[JENKINS-68293] Login and signup page refresh"
+    message: |-
+      Refresh the login and signup page.
+
+  - type: rfe
+    category: rfe
+    pull: 6473
+    issue: 68284
+    authors:
+      - timja
+    pr_title: "[JENKINS-68284] Modernise log recorder pages"
+    message: |-
+      Modernise log recorder pages.
+
+  - type: rfe
+    category: rfe
+    pull: 6295
+    authors:
+      - janfaracik
+    pr_title: 'Update form bottom button bar appearance '
+    message: |-
+      Update the appearance of the button bar at the bottom of forms.
+
+  - type: rfe
+    category: rfe
+    pull: 6273
+    authors:
+      - janfaracik
+    pr_title: "Update Manage Jenkins icons plugins icon now shows how many updates\
+      \ are available"
+    message: |-
+      Update "Manage Jenkins" icons and controls.
+      The plugins icon now shows how many updates are available.
+
+  - type: rfe
+    category: rfe
+    pull: 6229
+    authors:
+      - NotMyFault
+    pr_title: Use modern icons in workspace view of a job
+    message: |-
+      Use modern icons at the workspace view.
+
+  - type: rfe
+    category: rfe
+    pull: 6307
+    authors:
+      - janfaracik
+    pr_title: Replace Tango icons with Jenkins Symbols
+    message: |-
+      Update icons.
```

Comment:
These can probably be collapsed into one combined entry, like we did in the past for https://github.com/jenkins-infra/jenkins.io/blob/851e0be5dd5315c36f71f46f0fe59e51ee0b8825/content/_data/changelogs/lts.yml#L4018-L4039, https://github.com/jenkins-infra/jenkins.io/blob/851e0be5dd5315c36f71f46f0fe59e51ee0b8825/content/_data/changelogs/lts.yml#L4096-L4119, or https://github.com/jenkins-infra/jenkins.io/blob/851e0be5dd5315c36f71f46f0fe59e51ee0b8825/content/_data/changelogs/lts.yml#L6865-L6883